UIKit/Tasks/NavigationItem+Extension/NavigationItem.swift

```diff
@@ -0,0 +1,40 @@
+// NavigationItem.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Расширение для кнопки навигейшинконтроллера
+extension CoffeeSelectionViewController {
+    func setNavigationItemLeft() -> UIBarButtonItem {
```

Comment:
```suggestion
    func setNavigationItemLeft() -> UIBarButtonItem {
```
дока к методам см важное